lib/test.gi

```diff
@@ -371,7 +373,10 @@ end);
 ##  and that file is written with the same input and comment lines as
 ##  <Arg>fname</Arg> but the output substituted by the newly generated
 ##  version; if it is bound to <K>true</K>, then this is treated as if
-##  it was bound to <Arg>fname</Arg> (default is <K>false</K>).</Item>
+##  it was bound to <Arg>fname</Arg> (default is <K>false</K>). This is
+##  especially useful for long test files where <Ref Func="Test" />
+##  is sensible to line breaks and other whitespaces.
```

Comment:
I don't understand this sentence.

Reply:
Which part of it? It just gives the user a case in which this option can be helpful.

Reply:
I added a different explanation which should be clearer to understand for users.